tests/e2e/specs/modules/analytics/write-scope-requests.test.js

```diff
@@ -179,11 +179,11 @@ describe( 'Analytics write scope requests', () => {
 
 		// Click on confirm changes button and wait for permissions modal dialog.
 		await expect( page ).toClick( '.mdc-button', { text: /configure analytics/i } );
-		await page.waitForSelector( '.mdc-dialog__container' );
+		await page.waitForSelector( '.mdc-dialog--open' );
 
 		// Click on proceed button and wait for oauth request.
 		await Promise.all( [
-			expect( page ).toClick( '.mdc-dialog__actions .mdc-button', { text: /proceed/i } ),
+			expect( page ).toClick( '.mdc-dialog--open .mdc-button', { text: /proceed/i } ),
```

Comment:
What's the reason to change the selector here? I'd imagine it would be consistently failing if it's wrong, how is `.mdc-dialog-open` more reliable?

Reply:
I would expect it to fail consistently as well but that's the problem we're trying to fix here 😄 

This should be the selector we wait for anyways according to the docs for the `mdc-dialog--open` class:

> Indicates that the dialog is open and visible.

I'm guessing this instability may be related to a race condition related to instantiation/transitions in the underlying [`MDCDialog`](https://github.com/material-components/material-components-web/blob/v2.3.0/packages/mdc-dialog/README.md#javascript-instantiation)